Add files via upload

Schumiwk opened the pull request:
Added files for perseus device.

Schumiwk committed:
Add files via upload

gwolf2u commented on createjson.sh:
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```
please don't modify this file
modify locally only

gwolf2u commented on perseus.json:
```diff
@@ -0,0 +1,24 @@
+        "oem": "Xiaomi",
+        "device": "Perseus",
+        "filename": "crDroidAndroid-10.0-20200404-perseus-v6.4.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/<device codename>/<crversion>/crDroidAndroid-10.0-<date>-<device codename>-v<crversion>.zip/download",
```
use createjson.sh to generate this file
don't modify manually

Schumiwk committed:
Updated perseus.json

Hopefully it's better now, do understand that it's the first time I'm using github to do so.

gwolf2u commented:
createjson.sh is still changed 
please don't modify it
original file: https://github.com/crdroidandroid/android_vendor_crDroidOTA/blob/10.0/initial%20support/createjson.sh

Schumiwk committed:
Delete createjson.sh

Trying to make it work.
